modules/admin/config.rb

```diff
@@ -279,3 +347,4 @@ module Admin
   # rubocop: enable Metrics/BlockLength
 end
 # rubocop: enable Metrics/ModuleLength, Metrics/BlockNesting
+# vim: ts=2:sw=2:set expandtab
```

Comment:
please remove these

Reply:
I will do this after all the other requested changes